webdriver_wharf/app.py

```diff
@@ -172,18 +183,21 @@ def container_info(container):
     host = requesting_host()
     host_noport = host.split(':')[0]
 
-    return {
+    container_info = {
         'image_id': container.image_id,
         'checked_out': container.checked_out,
         'checkin_url': 'http://%s/checkin/%s' % (host, container.name),
         'renew_url': 'http://%s/renew/%s' % (host, container.name),
         'webdriver_port': container.webdriver_port,
         'webdriver_url': 'http://%s:%d/wd/hub' % (host_noport, container.webdriver_port),
-        'vnc_port': container.vnc_port,
-        'vnc_display': 'vnc://%s:%d' % (host_noport, container.vnc_port - 5900),
-        'http_port': container.http_port,
-        'fileviewer_url': 'http://%s:%d/' % (host_noport, container.http_port),
     }
+    if vnc_enabled:
+        container_info['vnc_port'] = container.vnc_port
+        container_info['vnc_display'] = 'vnc://%s:%d' % (host_noport, container.vnc_port - 5900)
```

Comment:
The problem isn't that vnc_display was in docs, the problem is that *because* vnc_display was in the docs, it's part of wharf's interface and must remain.